flydra_fly[] flies
float32 latency
